src/lib/utils.ts

```diff
@@ -13,6 +13,16 @@ export function isBuffer(obj: any) {
   );
 }
 
+/**
+ * Converts GitHub blob URLs to raw URLs
+ */
+export function normalizeUrl(url: string) {
```

Comment:
I kept this function name pretty generic in case we wanted it to do more in the future!